src/main/java/cms/gov/madie/measure/repositories/MeasureRepository.java

```diff
@@ -13,7 +13,11 @@
 public interface MeasureRepository extends MongoRepository<Measure, String> {
   Optional<Measure> findByCqlLibraryName(String cqlLibraryName);
 
-  Page<Measure> findAllByCreatedBy(String user, Pageable page);
+  Optional<Measure> findByIdAndActive(String id, Boolean active);
```

Comment:
I realize Spring is doing some lifting here to generate the mongo query, so if this suggestion introduces undue complexity, let me know.

Assuming `active` will most often be `true`, how about refactoring this method to default it to `true` or wrap it with another method that does so?

E.g. ```Optional<Measure> findByIsAndIsActive(String Id)```